Composer/packages/client/src/pages/botProject/adapters/AdapterSettings.tsx

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { useState, Fragment } from 'react';
+import formatMessage from 'format-message';
+import { useRecoilValue } from 'recoil';
+import { BotSchemas } from '@bfc/shared';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { TooltipHost, DirectionalHint } from 'office-ui-fabric-react/lib/Tooltip';
+import { SharedColors } from '@uifabric/fluent-theme';
+
+import { schemasState, settingsState, dispatcherState } from '../../../recoilModel';
+import { CollapsableWrapper } from '../../../components/CollapsableWrapper';
+import { title, subtitle, sectionHeader, tableRow, tableRowItem, tableColumnHeader } from '../styles';
+import { JSONSchema7 } from '../../../../../types';
+
+import AdapterModal, { AdapterRecord } from './AdapterModal';
+
+//////////
+
+type Props = {
+  projectId: string;
+};
+
+const AdapterSettings = (props: Props) => {
+  const { projectId } = props;
+
+  const schemas = useRecoilValue<BotSchemas>(schemasState(projectId));
+  const currentSettings = useRecoilValue(settingsState(projectId));
+  const { setSettings } = useRecoilValue(dispatcherState);
+  const adapters: AdapterRecord[] = currentSettings.adapters ?? [];
+
+  const { definitions: schemaDefinitions } = schemas?.default ?? {};
```

Comment:
```suggestion
  const { definitions: schemaDefinitions } = schemas?.sdk ?? {};
```

Reply:
Another option would be this:

```suggestion
  const { definitions: schemaDefinitions } = schemas?.sdk ?? schemas?.default ?? {};
```

It depends on the contract for schemas in the composer state. Is default schemas guaranteed to always exist? if yes we shoudl default to it rather than to `{}`.